src/llama_stack_client/lib/cli/inference/inference.py

```diff
@@ -31,7 +31,7 @@ def chat_completion(ctx, message: str, stream: bool, model_id: Optional[str]):
     console = Console()
 
     if not model_id:
-        available_models = [model.identifier for model in client.models.list()]
+        available_models = [model.identifier for model in client.models.list() if model.model_type == "llm"]
```

Comment:
dont we have enums for these model types? can we use them instead of string literals?